cogs/smallfunctions.py

```diff
@@ -0,0 +1,43 @@
+import discord
+import random
+import #new file storing priv variables
+from discord.ext import commands
+
+# Class responsible for all the small commands run by server members.
+class SmallFunctions(commands.cog):
+
+# Command function for randomized coinflip. Will return a heads or tails.
+    @commands.command()
+    async def random_coinflip(self, ctx):
+        heads = 1#store in diff file as const variables
+        tails = 0#store in diff file as const variables
+        rand_cf = random.randint(0,1)
+        if (rand_cf == heads):
```

Comment:
python doesn't need/shouldn't use `( )` in if statements. can just do `if rand_cf == heads`

Reply:
gotcha thank you